### ISSUE TITLE
Improve global search in start page

### ISSUE THREAD

**Issue by olensmar** [kubeshop/monokle#3996]
This currently only searches projects - although the hint suggests otherwise - suggest we 
- change the hint to "Find repositories and clusters"
- change the search to find projects and cluster contexts (it currently only finds projects)
- cluster would be shown as "Cluster <name>" - as opposed to the current "Project  <name>"
- selecting a cluster would connect to that directly and show the dashboard

wdyt @julioramosest @ayushgml @devcatalin ?

**Comment by ayushgml** [comment]
- Although I agreed with @julioramosest regarding the global search to include settings in the global search. I don't feel it is really needed at this point because the current settings panel does a good job in guiding the user through the Global and default project settings. Additionally, there are not much settings that user finds difficult to find. Settings search fits better when the user has difficulty finding the right setting. For example, when you're searching a setting on your laptop(OS has tons of settings!). 
 - I think repositories and clusters search would be better and optimal because when having many clusters, it would be easier for user to type the cluster -> click enter -> cluster is connected and dashboard is opened.
 - I think including the documentation would be better in efficient troubleshooting by the user. We should include it if we can.

**Comment by olensmar** [comment]
thanks @ayushgml - let's wait with searching the documentation since I don't think we really have that BE infrastructure in place, and the documentation needs an overhaul for it to be valuable - it's unfortunately lagging behind quite a lot..

**Comment by ayushgml** [comment]
Agreed @olensmar! Now let's hear @devcatalin and @julioramosest views.

**Comment by devcatalin** [comment]
I like the idea of including the cluster contexts in the search! It's a nice touch and could be helpful
